docs/user-guide.rst

```diff
@@ -581,6 +581,10 @@ Common build options
    1 (do save and restore). 0 is the default. An SPD may set this to 1 if it
    wants the timer registers to be saved and restored.
 
+- ``OVERRIDE_LIBC``: This option allows platforms to remove lib/libc from the
+   final image. It can take either 0 (include) or 1 (remove). The default
```

Comment:
Suggest: "This option allows platforms to override the default libc for the BL image. It can take either 0 (include) or 1 (roverride)"